frame/transaction-storage/README.md

```diff
@@ -0,0 +1,8 @@
+# Transaction Storage Pallet
+
+Indexes transactions and manages storage proofs.
+# Transaction Storage Pallet
+
+Indexes transactions and manages storage proofs.
+
```

Comment:
I know I'm late to the party, but it would be very cool to get a bit more of a high level description of the pallet.